packages/@o3r/third-party/schematics/iframe-to-component/index.ts

```diff
@@ -126,8 +128,8 @@ export function ngAddIframeFn(options: NgAddIframeSchematicsSchema): Rule {
 
                   /* eslint-disable indent */
                   const propertiesToAdd = generateClassElementsFromString(`
-                    @ViewChild('frame') private frame: ElementRef<HTMLIFrameElement>;
-                    private bridge: IframeBridge;
+                    @ViewChild('frame') private frame!: ElementRef<HTMLIFrameElement>;
```

Comment:
Could we use [viewChild](https://angular.dev/api/core/viewChild#) instead, to not have a dangerous `!`